src/server/search/aggregator.h

```diff
@@ -19,10 +20,16 @@ namespace dfly::aggregate {
 using Value = ::dfly::search::SortableValue;
 using DocValues = absl::flat_hash_map<std::string, Value>;  // documents sent through the pipeline
 
-// TODO: Replace DocValues with compact linear search map instead of hash map
+struct PipelineResult {
+  // Values to be passed to the next step
+  // TODO: Replace DocValues with compact linear search map instead of hash map
```

Comment:
is this TODO relevant? I do not see a hash map here

Reply:
Is relevant but I moved it to the wrong place. It relates to the line `using DocValues = absl::flat_hash_map<std::string, Value>;`. Was added [here](https://github.com/dragonflydb/dragonfly/pull/2413/files#diff-446ca7a8607d8aef6dd49edb8246b10c8ced16685508ad313c7b069d2b5024f1R22)